styles/style-a.css

```diff
@@ -1,3 +1,54 @@
 body {

```

Comment:
In general, it's best to start with the highest level element tags like body and then go to ids and then classes in terms of priority in the css file.